src/humancellatlas/data/metadata/api.py

```diff
@@ -582,10 +583,12 @@ class ImagingPreparationProtocol(Protocol):
     pass
 
 
-@dataclass
+@dataclass(init=False)
 class ManifestEntry:
+    json: MutableMapping = field(repr=False)  # should be MutableJSON
     content_type: str
     crc32c: str
+    target_id: str  # only populated in V2
```

Comment:
Why are we adding 
```suggestion
    target_id: Optional[str]  # only populated in v2
```

Reply:
But why are we adding `json` if we're also adding this?

If we continued this pattern, we'd have to add an optional property for every special-cased piece of data. That's what we wanted to avoid by adding `json` in the first place.

Reply:
Changed the initializer so this is no longer needed.